frontends/web/src/api/exchanges.ts

```diff
@@ -34,31 +27,35 @@ export type ExchangeRegion = {
   isPocketEnabled: boolean;
 }
 
+export const getExchangesByRegion = (code: string) => {
+  return (): Promise<ExchangeRegionList> => {
+    return apiGet(`exchange/by-region/${code}`);
+  };
+};
+
 export type ExchangeDeal = {
   fee: number;
   payment: 'card' | 'bank-transfer';
   isFast: boolean;
+  //not populated by BE response, meant for FE manipulation.
+  isBestDeal?: boolean;
 }
 
 export type ExchangeDeals = {
-  pocket: ExchangeDeal[];
-  moonpay: ExchangeDeal[];
+  exchangeName: string;
+  deals: ExchangeDeal[];
+  //not populated by BE response, meant for FE manipulation.
+  supported?: boolean;
```

Comment:
Ideally this is not defined in `api/exchanges` maybe @shonsirsha can move it later to `routes/buy/somewhere`

Reply:
Removed